teamB/cheonsongin/routes/likeRouter.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const likeController = require('../controllers/likeController');
+
+const router = express.Router();
```

Comment:
mport 순서 컨벤션에 맞게 수정해주세요! 사소한 부분이라 놓치기 쉽지만 이런 컨벤션 준수들이 모여서 좀 더 가독성 좋은 코드가 됩니다

```suggestion
const express = require('express');
const router = express.Router();

const likeController = require('../controllers/likeController');

```